### PR TITLE
Doc update + release note

### PR DESCRIPTION
**Pull request by v-himkam**

### CODE REVIEW

File: 01-Setup and Getting started/Readme.md [hunk 1]
```diff
@@ -106,6 +106,16 @@ The CSV file and LOG file are generated under a org-specific sub-folder in the f
 E.g.  
 C:\Users\<UserName>\AppData\Local\Microsoft\AzSK.ADOLogs\Org_[yourOrganizationName]\20181218_103136_GADS
 
+## Customize location for scan reports
+The path for saving scan reports can be customized using below command:
```
Comment (abhaydaga) [inline 01-Setup and Getting started/Readme.md:110]: The location to save scan reports can now be customized using the command below:

File: 01-Setup and Getting started/Readme.md [hunk 2]
```diff
@@ -106,6 +106,16 @@ The CSV file and LOG file are generated under a org-specific sub-folder in the f
...
+```Powershell
+Set-AzSKADOUserPreference -OutputFolderPath '<Custom folder path>'
+```
+To reset the path for saving scan reports to default location use below command:
```
Comment (abhaydaga) [inline 01-Setup and Getting started/Readme.md:114]: same as above

File: 01-Setup and Getting started/Readme.md (hunk 1 above)
Comment (v-himkam) [reply 01-Setup and Getting started/Readme.md]: updated the statement

File: 01-Setup and Getting started/Readme.md (hunk 2 above)
Comment (v-himkam) [reply 01-Setup and Getting started/Readme.md]: updated the statement

**Pull request merged**